src/templates/newengland.html

```diff
@@ -0,0 +1,60 @@
+{% import "components/css.html" as css %}
+{% extends "base.html" %}
+{% block title%}New England Mailing List{% endblock %}
+{% block content %}
+  <div class="container">
+    <div class="card">
+      <div class="card-body">
+        <div class="card-title">
+          <h1>New England Mailing List</h1>
+        </div>
+        <div class="card-text">
+          <p>Note: this mailing list is operated by local Delegates, not by CubingUSA. CubingUSA is not responsible for this mailing list, and there is no guarantee that every competition will be posted here.</p>
+          <!-- Begin MailChimp Signup Form -->
+            <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
+            <style type="text/css">
+              #mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; }
+              /* Add your own MailChimp form style overrides in your site stylesheet or in this style block.
```

Comment:
Do we want to follow this advice?

Reply:
To clarify, this is copy and pasted from MailChimp-generated code for the form. As for the style, I'm open to style suggestions but it actually fits in with the rest of the page surprisingly nicely (screenshot below)

![newengland-mailing-list-screenshot](https://user-images.githubusercontent.com/3170853/42776766-0b8cf738-8906-11e8-8bc9-cec0f31d60c7.png)